src/System.Net.WebSockets.Client/src/System/Net/WebSockets/WebSocketHandle.Managed.cs

```diff
@@ -271,6 +290,30 @@ private static byte[] BuildRequestHeader(Uri uri, ClientWebSocketOptions options
             }
         }
 
+        /// <summary>
+        /// Given a &quot;Host:Port&quot; string return only the &quot;Host&quot;
+        /// </summary>
+        /// <param name="hostAndPort">The host and optional port value (e.g. &quot;Host:Port&quot;).</param>
```

Comment:
No problem, I could have sworn at some point in the past VS complained about invalid XML when I did that.